### PR TITLE
Move abbreviations to a different file

### PR DESCRIPTION
**Pull request by zzarcon**
I think now the index file, which is the main one will lock more clear

### CODE REVIEW

File: lib/abbreviations.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+module.exports = function(commands) {
+  var abbreviations = abbrev(expandNames(commands));
+  
+  abbreviations.i = 'install';
+  abbreviations.rm = 'uninstall';
+  abbreviations.unlink = 'uninstall';
+  abbreviations.ls = 'list';
+
+  return abbreviations;
+};
```
Comment (sheerun) [inline lib/abbreviations.js]: No newline at the end :) Please configure your editor

Comment (zzarcon) [inline lib/abbreviations.js]: Ups, good catch

**Comment by zzarcon** [comment]
@sheerun what do you think now? ^^

**Comment by sheerun** [comment]
It looks really good :) Could you squash the commits and force push as the last step?

**Comment by zzarcon** [comment]
@sheerun done :dancer:

**Comment by sheerun** [comment]
Great, thank you!

**Comment by faceleg** [comment]
Getting a lot of activity recently, awsome!

**Comment by zzarcon** [comment]
@faceleg yeah!

**Pull request merged**